CodePush.js

```diff
@@ -110,6 +123,23 @@ function log(message) {
   console.log(`[CodePush] ${message}`)
 }
 
+async function notifyApplicationReady() {
+  await NativeCodePush.notifyApplicationReady();
+  const statusReport = await NativeCodePush.getNewStatusReport();
+  if (statusReport) {
+    const config = await getConfiguration();
+    if (statusReport.appVersion) {
+      config.appVersion = statusReport.appVersion;
```

Comment:
Is this necessary? Wouldn't the status report already include the same app version that would be part of the config?